cms/jinja2/templates/base_page.html

```diff
@@ -35,8 +35,14 @@
     {# Changes the location of the <main> tag from the base template - ensures it surrounds the header area #}
     {# This change should be adopted by the design system soon #}
     <main id="main-content">
-        {# This will contain the breadcrumbs on individual templates #}
+        {# This block should be overriden on individual page templates and include the breadcrumbs and page heading. This is default content while the build is in progress. #}
```

Comment:
https://github.com/ONSdigital/dis-wagtail/blob/main/cms/jinja2/templates/pages/release_calendar/release_calendar_page.html now shows two titles.

Reply:
That should now be resolved.